### PR TITLE
Update: `joncampbell123.DOSBox-X` version 0.83.15

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/20746)

### CODE REVIEW

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
On a x64 machine:
image

**Comment by vedantmgoyal2009** [comment]
What to do?

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Previous manifests, had `/verysilent` as a switch, and by default winget-cli runs it with `/silent`

**Comment by vedantmgoyal2009** [comment]
Ok

File: manifests/j/joncampbell123/DOSBox-X/0.83.15/joncampbell123.DOSBox-X.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+﻿# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+
+PackageIdentifier: joncampbell123.DOSBox-X
+PackageVersion: 0.83.15
+PackageLocale: en-US
+Publisher: joncampbell123
```
Comment (ItzLevvie) [inline manifests/j/joncampbell123/DOSBox-X/0.83.15/joncampbell123.DOSBox-X.locale.en-US.yaml:7]:
```suggestion
Publisher: joncampbell123 [DOSBox-X Team]
```

File: manifests/j/joncampbell123/DOSBox-X/0.83.15/joncampbell123.DOSBox-X.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: inno
+  Scope: user
+  InstallerUrl: https://github.com/joncampbell123/dosbox-x/releases/download/dosbox-x-v0.83.15/dosbox-x-windows-0.83.15-setup.exe
+  InstallerSwitches:
+   Silent: /VERYSILENT
+   SilentWithProgress: /VERYSILENT
```
Comment (ItzLevvie) [inline manifests/j/joncampbell123/DOSBox-X/0.83.15/joncampbell123.DOSBox-X.installer.yaml:21]:
```suggestion
   Silent: /VERYSILENT /CURRENTUSER
   SilentWithProgress: /VERYSILENT /CURRENTUSER
```

File: manifests/j/joncampbell123/DOSBox-X/0.83.15/joncampbell123.DOSBox-X.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  UpgradeBehavior: install
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: inno
+  Scope: machine
+  InstallerUrl: https://github.com/joncampbell123/dosbox-x/releases/download/dosbox-x-v0.83.15/dosbox-x-windows-0.83.15-setup-allusers.exe
+  InstallerSha256: F9905231D310C6BD22B6CE8E3BA942760992CCA4883CFEEAB4E34C8F53470357
+  InstallerSwitches:
+   Silent: /VERYSILENT
+   SilentWithProgress: /VERYSILENT
```
Comment (ItzLevvie) [inline manifests/j/joncampbell123/DOSBox-X/0.83.15/joncampbell123.DOSBox-X.installer.yaml:32]:
```suggestion
   Silent: /VERYSILENT /ALLUSERS
   SilentWithProgress: /VERYSILENT /ALLUSERS
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
image

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**